### DISCUSSION TITLE
Fixed service fee already included in ticket price

### DISCUSSION THREAD

**Discussion by mvanderkruk** [pretix/pretix#3379] [Ideas]
In the EU it is not allowed to add a service fee to the listed price, the price listed has to be including the service fee.
I would like the service fees plugin to have a function where it is possible to specify a fixed service fee on a product which is already included in the displayed price in the shop.
This also has to be displayed separately on the invoice as the organiser does not have to pay a fee to for example BUMA on the service fee, but does have to pay on the product price itself.
Example as to how it is displayed on a different ticketshop:
![image](https://github.com/pretix/pretix/assets/2444979/faf3c1f4-af5f-4dc2-b2f2-9207989f4b2a)

**Comment by raphaelm** [comment]
This would not be possible through the service fee module or through the fee system of pretix entirely. You could do it through "bundled products" if you need to

**Reply by fabm3n** [reply]
But even if I use the bundled products for a ticket fee, it is not displayed separately on the overview shop site. It only displays the fees in the cart.

I also tried the service fee plugin, but it's not working at all. The Plugin shows installed in the update checker, but it does not appear under the event settings. When using the path settings/servicefees/ it shows the configuration but if I set a fee, it will not be calculated in the shop.
Maybe the plugin is too old?

**Reply by raphaelm** [reply]
The plugin works on our system at least

**Reply by fabm3n** [reply]
Just did a completely new installation in a LXC Container, and it didn't show up in the Event Settings:
<img width="299" alt="image" src="https://github.com/pretix/pretix/assets/11352551/0db8e667-cdee-4265-a7e6-91239ca66378">

With the url i can access the settings page:
<img width="1710" alt="image" src="https://github.com/pretix/pretix/assets/11352551/30620471-4d82-44df-a436-2049dde230c7">

Update checker:
<img width="1405" alt="image" src="https://github.com/pretix/pretix/assets/11352551/67813dbf-64fd-42a7-b949-da0970f99bb4">


Maybe you have a newer version internally?

**Reply by fabm3n** [reply]
I started to debug the plugin locally. I added more logging in the signals.py of the plugin and in general the signal.py will be executed. But none of the loggers in the receivers will be executed. I also tried to add a testing receiver:
```python
@receiver(front_page_bottom, dispatch_uid="service_fee_testinghtml")
def testinghtml(sender, **kwargs):
    logger.info('inside testinghtml')
    return('<p>Bottom by testinghtml</p>')
```
But even this receiver will never be executed.

**Reply by pc-coholic** [reply]
Ok, stupid question... But did you enable the plugin within the event settings -> plugins after installing it?

**Reply by fabm3n** [reply]
> Ok, stupid question... But did you enable the plugin within the event settings -> plugins after installing it?

You got me… Totally forgot to enable it on event level…
But I learned a lot of how plugins are working :D
